README.md

```diff
@@ -126,8 +126,8 @@ Note that these builds may be unstable since the develop branch is a work in pro
 
 To get nightly downloads, please
 [click here](https://github.com/logisim-evolution/logisim-evolution/actions/workflows/nightly.yml)
-and browse to the last successful run of `Build` worker, which should be on top. Note that due to Github internals,
-all files are provided as ZIP archives. You must unzip the downloaded file to get the package for installation.
+and browse to the last successful run of `Nightly` worker, which should be on top. Note that due to Github internals,
+all files are provided as ZIP archives. You must unzip downloaded file to get the installation package to use.
```

Comment:
Please replace the "the" between unzip and downloaded. It is an English thing.

The change at the end of the sentence is cleaner than what we had before. But I'd suggest it would be even cleaner to delete "to use" and put the period after package, since using it is implied in the process. That is not an "English thing," so your choice.

Reply:
Fixed, thanks for checking.